be/src/olap/predicate_creator.h

```diff
@@ -49,7 +49,7 @@ class PredicateCreator {
 template <PrimitiveType Type, PredicateType PT, typename ConditionType>
 class IntegerPredicateCreator : public PredicateCreator<ConditionType> {
 public:
```

Comment:
maybe not right here. because many predicate parse the value from string, and olap scannode convert the predicate  value to a string and predicate creator parse the value from string.
If you do not change olap scannode code, then I think you could not parse the value correctly.
I think, maybe we should delete predicate creator.